index.html

```diff
@@ -98,89 +99,95 @@ <h2>The Finance and Investment club of NIT Trichy.</h2>
 
         <div class="section-title">
           <h2>About Us</h2>
-          <p>Magnam dolores commodi suscipit. Necessitatibus eius consequatur ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.</p>
+          <p>What started out as a WhatsApp Group among students interested in finance, ProfNITT is now the finance and investments club of NIT Trichy,
+             with more than 30 members.We aim to bring about financial literacy among students of NIT-Trichy, helping them make well-informed
+             monetary decisions.
+          </p>
         </div>
 
         <div class="row content">
           <div class="col-lg-6">
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-              magna aliqua.
-            </p>
+            <p>The club conducts several events throughout the year to engage with the student community such as</p>
             <ul>
-              <li><i class="ri-check-double-line"></i> Ullamco laboris nisi ut aliquip ex ea commodo consequat</li>
-              <li><i class="ri-check-double-line"></i> Duis aute irure dolor in reprehenderit in voluptate velit</li>
-              <li><i class="ri-check-double-line"></i> Ullamco laboris nisi ut aliquip ex ea commodo consequat</li>
+              <li><i class="ri-check-double-line"></i>Knowledge Sharing Sessions</li>
+              <li><i class="ri-check-double-line"></i>Guest Lectures and Webinars</li>
+              <li><i class="ri-check-double-line"></i>Workshops</li>
+              <li><i class="ri-check-double-line"></i>Case Solving Competitions</li>
             </ul>
           </div>
           <div class="col-lg-6 pt-4 pt-lg-0">
             <p>
-              Ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-              velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-              culpa qui officia deserunt mollit anim id est laborum.
+             The club also aims to empower its members by increasing their knowledge base in finance related domains, via various interactions,
+              discussions and industry projects brought with the help of our strong alumni network.
             </p>
             <a href="#" class="btn-learn-more">Learn More</a>
           </div>
         </div>
 
       </div>
-    </section><!-- End About Us Section -->
+    </section>
+
+    <!-- End About Us Section -->
 
-    <!-- ======= Services Section ======= -->
+    <!-- ======= Events Section ======= -->

```

Comment:
Need to change the corresponding closing comment at line 197

Reply:
cool will make it